infra/dlv/dlv-ctl.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.!/bin/bash
+
+SSH="ssh -o StrictHostKeyChecking=no"
+SCP="scp -o StrictHostKeyChecking=no"
+
+REMOTE_DLV_ATTACH=/usr/local/bin/dlv-attach-headless.sh
+REMOTE_DLV_DETACH=/usr/local/bin/dlv-detach-headless.sh
+
+function usage() {
+    echo "Usage: $0 -h vch-address [-a/-d] -p password [attach/detach] target" >&2
+    echo "Valid targets are: "
+    echo "    vic-init"
+    echo "    vic-admin"
+    echo "    docker-engine"
+    echo "    port-layer"
+    echo "    virtual-kubelet"
```

Comment:
Given that this is merging to `master`, it doesn't seem like we should mention `virtual-kublet`; that can be added on the feature branch.